distribution/ddf-common/src/main/resources/security/default.policy

```diff
@@ -339,9 +339,11 @@ grant codeBase "file:/catalog-core-standardframework/platform-solr-server-standa
     permission java.io.FilePermission "${ddf.home.perm}etc${/}ws-security${/}server${/}-", "read";
 }
 
-grant codeBase "file:/catalog-plugin-videothumbnail" {
+grant codeBase "file:/catalog-plugin-videothumbnail/catalog-rest-endpoint/catalog-core-standardframework" {
     permission java.io.FilePermission "${ddf.home.perm}bin_third_party", "read,write";
     permission java.io.FilePermission "${ddf.home.perm}bin_third_party${/}-", "read,write,execute";
+    permission java.io.FilePermission "${ddf.home.perm}bin_third_party${/}ffmpeg", "read,write";
```

Comment:
:question: this really shouldn't be necessary. Did you find that the recursive permission on the parent directory wasn't sufficient?

Reply:
Probably not. I will remove it.